nfl_server/public/attribute_checkboxes.js

```diff
@@ -1,11 +1,12 @@
 // TODO: Uncomment a value to create a checkbox for it
 const allowedAttributes = {
 	qb: [ 'Cmp%', 'Yds', 'TD', 'Rate', 'W' ],
-	wr: [ 'Rec', 'Yds', 'Y/R', 'TD', 'Y/G' ]
+	wr: [ 'Rec', 'Yds', 'Y/R', 'TD', 'Y/G' ],

```

Comment:
We should expand on these both in the CSV and initialization so that it's: `Receptions, Yards, Yards/Reception` etc. just to make it a little clearer